idaes/gas_solid_contactors/unit_models/fixed_bed_0D.py

```diff
@@ -217,13 +217,16 @@ def solids_material_holdup_constraints(b, t, j):
                     self.solids_material_holdup,
                     wrt=self.flowsheet().config.time,
                     doc="Solids material accumulation",
-                    units=units_meta_solid('mass')/units_meta_solid('time'))
+                    units=units_meta_solid('mass'))
+        # should be mass/time, to get around DAE limitations var is
+        # divided by time units when used
```

Comment:
Pyomo does not currently support units for ContinuousSet objects, and @andrewlee94 has opened an issue requesting this feature. Since we cannot specify mass/time units here but the numerical calculations are still correct, the accumulation variables are manually corrected for units in the accumulation constraints.

Reply:
@bpaul4 I am not sure that this is correct. In the 1D control volume, we assign the correct units to the `DerivativeVar` (see line 424 in CV1D). I am not sure how we made this work, but it does.

Reply:
It appears line 268 in the 0D control volume does something similar, although Pyomo threw an InconsistentUnits error when I attempted to set expected units on the derivative variable itself. An internal constraint defined the discretization as accumulation = Δ(mass)/Δ(time) with units [m/s] = [m]/[dimensionless since ContinuousSets are not passed with units].

Reply:
Okay, I think I see. The problem is that the "Delta t" constants in the time discretization equation don't have units. It's not a problem for the space discretization equations because our length domains are normalized (dimensionless).